cdk/README.md

```diff
@@ -1,6 +1,6 @@
 # Example CDK implementation - Amazon ECS on AWS Fargate with Consul Connect service mesh
 
-In this example we'll configure one Consul server in VPC with TLS and gossip encryption enabled. Using [AWS CDK ECS service extension for Consul](https://github.com/aws-ia/ecs-consul-mesh-extension), we'll create and deploy the application stack that will launch ECS cluster with sample `greeter` application connected to Consul service mesh.
+In this example we'll configure one Consul server in VPC with TLS and gossip encryption enabled. Using [AWS CDK ECS service extension for Consul](https://github.com/aws-ia/ecs-consul-mesh-extension), this sample app will will launch ECS cluster with sample `greeter` application connected to Consul service mesh.
```

Comment:
nit: there are 2 _will_ one after the other. We can remove one. 

I am still thinking if **greeter application connected to Consul Service Mesh** is  apt to say? 
Or we can say - this sample app will launch ECS cluster with sample `greeter` application where all the services in the application will be talking to each other over consul service mesh.

Reply:
thanks, I'll incorporate this.

Reply:
updated readme in [2323adc](https://github.com/aws-samples/amazon-ecs-fargate-consul-connect-example/pull/16/commits/2323adc2ca5f8e838f13de6a8e99377784dd2f3b)